client/src/components/SliderCard.jsx

```diff
@@ -3,21 +3,55 @@ import ReactDOM from 'react-dom';
 import styled from 'styled-components';
 
 const StyledCard = styled.div`
+  flex: 1 0 25%;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 5fr 1fr 0.5fr 0.5fr;
+  gap: 0px 0px;
+  grid-template-areas:
+    "pic"
+    "title"
+    "pricing"
+    "icons";
+  margin: 10px;
+  width: 350px;
+`;
+
+const CardPictureStyle = styled.div`
   display: flex;
-  flex-direction: column;
   justify-content: center;
+  grid-area: pic;
+  overflow: hidden;
   background-color: #f5f5f5;
-  height: auto;
-  width: 25%;
   border-style: none;
   border-radius: 1rem;
-  sizes: 723px;
+  object-fit: cover;
 `;
 
-const StyledText = styled.div`
-  display: flex;
-  justify-content: flex-start;
-  padding: 0 1.6rem 0;
+const StyleImg = styled.img`
+  max-width: auto;
```

Comment:
@masoncpott I have never used this before. What's the difference between max-width and width?

Reply:
I think max-width helps with resizing when a window gets smaller?